src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.spec.ts

```diff
@@ -44,6 +45,7 @@ describe('DotEditPageMainComponent', () => {
                 providers: [
                     { provide: DotMessageService, useValue: messageServiceMock },
                     { provide: PageViewService, useClass: PageViewServiceMock },
+                    DotLicenseService
```

Comment:
why?

Reply:
mock it ? or add DotLicenseService to DotTestBed ?

Reply:
DotTestBed